src/swipes.py

```diff
@@ -0,0 +1,304 @@
+    # sum together the swipes of rows with same timeblock/eatery makeup
+    df = df.groupby(ISOLATE_SWIPES).sum().reset_index()
+    # count how many days of the same timeblock/eatery combo we have --> used for counter column
+    df = df.groupby(ISOLATE_DATE).count().reset_index()
+    df = df.rename(index=str, columns={'date': 'counter'})
+    # sum together swipes and counters for rows with the same timeblock and location
+    df = df.groupby(ISOLATE_COUNTER_SWIPES).sum().reset_index()
+    df = calculate_wait_times(df)
+    df = df.sort_values(by=['location', 'weekday'])
+    output_path = '{}{}'.format(EATERY_DATA_PATH, output_file)
+    df.to_csv(output_path, header=True, index=False)
+    return output_path
+  except Exception as e:
+    print('Failed at sort_by_timeblock')
+    print('Data update failed:', e)
+
+
+def sort_by_day(input_file_path, output_file='daily-averages.csv'):
+  """
+  Sorts and runs average swipe/time calculations.
+  Saves to a csv file.  Currently not used in main code, but allows for future use of data
+  daily-averages.csv - the daily swipe averages
+    weekday: weekday of swipe event, string (ex: monday, tuesday, etc.)
+    in_session: whether or not school is on a break, boolean
+    location: eatery of swipe event, string (some locations have different names than dining.now's)
+    swipes: number of swipes, int
+    counter: number of events within this timeblock (used to calculate average), int
+    average: average number of swipes in this timeblock, float (2 decimals)
+  """
+  try:
+    df = pd.read_csv(input_file_path)
+    df = df.drop(columns=['start_time', 'end_time', BRB_ONLY, DINING_HALL, TRILLIUM])
+    # sum together the swipes of rows with same day/eatery makeup
+    df = df.groupby(['date', 'session_type', 'weekday', 'location']).sum().reset_index()
+    # count the number of times a location/date pair occur
+    df = df.groupby(['weekday', 'location', 'swipes', 'session_type']).count().reset_index()
+    df = df.rename(index=str, columns={'date': 'counter'})
+    # sum swipes and counters for rows with the same weekday and location
+    df = df.groupby(['weekday', 'location', 'session_type']).sum().reset_index()
+    df['average'] = np.around(np.divide(df['swipes'], df['counter']), decimals=2)
+    df = df.sort_values(by=['location', 'weekday'])
+    df.to_csv('{}{}'.format(EATERY_DATA_PATH, output_file), header=True, index=False)
+  except Exception as e:
+    print('Faled at sort_by_day')
+    print('Data update failed:', e)
+
+
+def export_data(file_path):
+  """
+  Transforms our tabular data into custom objects to be placed in Eatery objects
+  """
+  global breaks
+  global weekdays
+  try:
+    df = pd.read_csv(file_path)
+    data = {}
+    today = date.today()
+    session_type = sort_session_type(today, breaks)
+    print(weekdays[today.weekday()])
+    for location in df['location'].unique():
+      true_location = LOCATION_NAMES[location]['name']
```

Comment:
`true_location`? what makes it a "true" location?

Reply:
like the actual name that makes sense (ex: 'Duffield' --> 'Mattin's Cafe')